### PR TITLE
New Features, Bug Fixes, and Updates!

### PR DESCRIPTION
**Pull request by Grimsbain**
nChat
1) Added option to show/hide the quick join button.

nCore
1) Modified the "On Demand" text in the addon list window.
2) Added a module that modifies the dressroom windows to allow the user to remove all displayed gear.
3) Added a module that displays an alert when a vignette is detected.
4) Updated Skada skin.

nMinimap
1) Fixed mail icon.
2) Added check for lagHome to prevent the memory tooltip from showing when the game is first loading.

nTooltip
1) Added some new tooltip types.
2) Fixed ICON_LIST sometimes returning nil.
3) Changed from UnitSelectionColor to GameTooltip_UnitColor to fix coloring in some cases.

oUF_Neav
1) Updated HealthPrediction to handle new oUF elements.
2) Updated FormatValue function to be more exact and be able to handle larger numbers.
3) Small fix to the "Set Focus" tab.
4) Removed unused combo point textures.

oUF_NeavRaid
1) Dropped smartthreat module for default oUF element.
2) Updated HealthPrediction to handle new oUF elements.
3) Updated the Freebgrid_AuraList to include new content.

### CODE REVIEW

**Comment by renstrom** [comment]
That's a lot of fixes, cheers!

File: Interface/AddOns/nCore/core.lua [hunk 1]
```diff
@@ -26,6 +26,10 @@ SlashCmdList['RELOADUI'] = function()
 end
 SLASH_RELOADUI1 = '/rl'
 
+if ( AddonList ) then
+    _G['ADDON_DEMAND_LOADED'] = "On Demand";
+end
```
Comment (renstrom) [inline Interface/AddOns/nCore/core.lua]: Any risk for taint by changing `ADDON_DEMAND_LOADED`?

Comment (Grimsbain) [inline Interface/AddOns/nCore/core.lua]: I have had it set like that for awhile now and I haven't ran into any issues. It isn't a major issue just a small tweak so it is up to you.

**Comment by renstrom** [comment]
Does this fix these oUF_NeavRaid issues mentioned on WoWI?

438062wowscrnshot101816232802

**Comment by Grimsbain** [comment]
I think I got that one sorted out but I'm not totally sure about the healing absorb bars atm. It is hard to test for them.
wowscrnshot_071917_160352

**Comment by Kkthnx** [comment]
There was no issue, to begin with. They were/are using an older version. You can tell by the unit frames. I tested earlier today and didn't have 1 bit of an issue.

**Comment by renstrom** [comment]
Yeah one thing I noticed is that absorbs look a bit out of place when you have full health:

untitled

**Pull request merged**